src/xcode/ENA/ENA/Source/Services/HealthCertificateValidationService/Model/ValidationOnboardedCountriesCache.swift

```diff
@@ -0,0 +1,43 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct ValidationOnboardedCountriesCache: Codable, Equatable {
+    
+    // MARK: - Init
+    
+    init(
+        onboardedCountries: [Country],
+        lastOnboardedCountriesETag: String
+    ) {
+        self.onboardedCountries = onboardedCountries
+        self.lastOnboardedCountriesETag = lastOnboardedCountriesETag
+    }
+    
+    // MARK: - Overrides
+    
+    // MARK: - Protocol Codable
```

Comment:
I guess implementing codable is not needed here, it should be inferred by the compiler.

Reply:
It is needed for the SecureStore.

Reply:
Yes right, codable is needed, but do you need to implement it? Isn't it enough to just add the protocol?